### PR TITLE
Tip about IPv4/IPv6 in listen()

### PR DESCRIPTION
**Pull request by sebdeckers**
Small hint to the reader about setting up their server for IPv6 and IPv4.

#### Checklist

- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ Fastify offers an easy platform that helps solve all of problems, and more.
 > })
 > ```
 >
+> Similarly, specify `::1` to accept only local connections via IPv6, or `::` to accept any connection on all IPv4 and IPv6 addresses simultaneously.
```
Comment (jsumners) [inline docs/Getting-Started.md]: The `::` may or _may not_ open the socket on all IPv4 addresses in addition to the IPv6 addresses. It is OS dependent according to the core Node documentation. Therefore, this statement is not complete.

Comment (sebdeckers) [reply docs/Getting-Started.md]: TIL, thanks for catching that.

File: docs/Getting-Started.md [hunk 2]
```diff
@@ -67,6 +67,8 @@ Fastify offers an easy platform that helps solve all of problems, and more.
 > })
 > ```
 >
+> Similarly, specify `::1` to accept only local connections via IPv6. Or specify `::` to accept connections on all IPv6 addresses, and, if the operating system supports it, also on all IPv4 addresses.
```
Comment (allevo) [inline docs/Getting-Started.md]: @jsumners Maybe here we can link the nodejs doc?
https://nodejs.org/docs/latest/api/net.html#net_server_listen_port_host_backlog_callback

Comment (jsumners) [reply docs/Getting-Started.md]: /shrug

I think the text covers it. People can do some research if they want to know more.

**Pull request merged**